docs/services/data/capture-sync.md

```diff
@@ -1,29 +1,37 @@
 ---
-title: "Configure Data Capture"
-linkTitle: "Data Capture"
-description: "Configure data capture to save data from components, services, or remote parts."
+title: "Data Capture and Sync"
+linkTitle: "Data Capture and Sync"
+description: "Configure the data manager to capture data from your components and services and sync it to the cloud."
 weight: 10
 type: "docs"
-tags: ["data management", "cloud", "sync"]
+tags: ["data management", "cloud", "sync", "capture"]
 icon: true
 images: ["/services/icons/data-capture.svg"]
 no_list: true
 aliases:
-  - "/services/data/capture/"
-  - "/data/capture/"
-  - "/build/micro-rdk/data_management/"
-# SME: Alexa Greenberg
+  - /services/data/capture/
+  - /data/capture/
+  - /build/micro-rdk/data_management/
+  - /services/data/capture/
+  - /services/data/cloud-sync/
+  - /data/cloud-sync/
 no_service: true
 ---
 
-The data management service captures data from one or more resources.
+The data management service captures data from one or more {{< glossary_tooltip term_id="resource" text="resources" >}} locally, and syncs it to cloud storage when a connection to the cloud is available.
+You can configure which data you want to capture, as well as the capture rate and the sync frequency.
+
+Data capture is frequently used with cloud sync.
+However, if you want to manage your machine's captured data yourself, you can enable only data capture without cloud sync.
 
 Get started with a quickstart guide or keep reading for more details.
 
 {{< cards >}}
 {{< card link="/get-started/collect-data/" class="green">}}
 {{< /cards >}}
 
+## Data capture details
+
```

Comment:
I think both this and the cloud sync details heading should go - we don't really want to separate data capture and sync mentally.